### ISSUE TITLE
problematic samples ?

### ISSUE THREAD

**Issue by colindaven** [SilasK/amplicon-seq-dada2#5]
Hi,
just trying this, the test and install worked fine. thanks.

It seems my sample table isn't ok despite using the supplied script.

It looks like this, I've tried removing many samples, removing SE read samples (public data), gzipping, gunzipping, adding _001.fastq, removing _001.fastq ...... no luck.

Are the names just too long or contain illegal chars ?

16S-dada2$ snakemake --cores 10

KeyError in line 5 of /ngsssd1/rcug/public_metagenome/soil_16S/16S-dada2/Snakefile:
'sampletable'
  File "/ngsssd1/rcug/public_metagenome/soil_16S/16S-dada2/Snakefile", line 5, in <module>

Thanks,
Colin

```
cat samples.tsv
        R2      R1
SRR12688636-16S-microbiome-001  /ngsssd1/rcug/public_metagenome/soil_16S/renamed/microbiome/SRR12688636_16S_microbiome_R2_001.fastq     /ngsssd1/rcug/public_metagenome/soil_16S/renamed/microbiome/SRR12688636_16S_microbiome_R1_001.fastq
SRR12688638-16S-microbiome-001  /ngsssd1/rcug/public_metagenome/soil_16S/renamed/microbiome/SRR12688638_16S_microbiome_R2_001.fastq     /ngsssd1/rcug/public_metagenome/soil_16S/renamed/microbiome/SRR12688638_16S_microbiome_R1_001.fastq
SRR12688640-16S-microbiome-001  /ngsssd1/rcug/public_metagenome/soil_16S/renamed/microbiome/SRR12688640_16S_microbiome_R2_001.fastq     /ngsssd1/rcug/public_metagenome/soil_16S/renamed/microbiome/SRR12688640_16S_microbiome_R1_001.fastq
SRR12688641-16S-microbiome-001  /ngsssd1/rcug/public_metagenome/soil_16S/renamed/microbiome/SRR12688641_16S_microbiome_R2_001.fastq     /ngsssd1/rcug/public_metagenome/soil_16S/renamed/microbiome/SRR12688641_16S_microbiome_R1_001.fastq
SRR12688642-16S-microbiome-001  /ngsssd1/rcug/public_metagenome/soil_16S/renamed/microbiome/SRR12688642_16S_microbiome_R2_001.fastq     /ngsssd1/rcug/public_metagenome/soil_16S/renamed/microbiome/SRR12688642_16S_microbiome_R1_001.fastq
SRR12688637-16S-microbiome-001  /ngsssd1/rcug/public_metagenome/soil_16S/renamed/microbiome/SRR12688637_16S_microbiome_R2_001.fastq     /ngsssd1/rcug/public_metagenome/soil_16S/renamed/microbiome/SRR12688637_16S_microbiome_R1_001.fastq

```

**Comment by SilasK** [comment]
Thank you for testing my pipeline. I fixed your problem in 50bf7f8

**Comment by colindaven** [comment]
Thanks, unfortunately that didn't fix it.

I corrected the R1 and the R2 columns in the file (first R1 then R2 using awk).

`awk -F "\t" -v OFS='\t' '{print $1,$3,$2}' samples.tsv > samples2.tsv
`

However there is now a dada2 threads issue... will comment separately
